README.md

```diff
@@ -1,3 +1,11 @@
+## Annoucement from Engineering @ Faire
+Hello mjml community, we at Faire are excited to announce we will be taking over maintaining the open-source miml-react project! 
+
+Over the past year, Faire has used miml-react extensively. This package allowed us to incorporate all the benefits of mjml without having to ask our developers to learn a new dev environment moving from web to email. As a result, we significantly reduced the dev effort for creating emails, while keeping email template issues to a minimum. 
+
+We saw so much benefit from miml-react, and we don’t want to see it go. That is why we want to continue to maintain the package, and even improve it with some of our own new ideas! For now you can check us out at LINK for a drop in replacement for miml-react. From there you can look forward to more improvements in the future.
```

Comment:
I have an update for this paragraph in our channel. Also the link to mjml-react is https://mjml.slack.com/archives/C043G87R7T9